citi-onboarding/EBM-Solu-es
# implementacao do rodape

###What I did:
Implementacao do rodape

How to test it
1)Open the terminal and go to the folder of this project
2)In terminal:
a) git checkout develop
b) git pull
c) git checkout feature/reportDemand
d) git pull origin feature/reportDemand
e) yarn install
f) code .
g) yarn start

In your browser go to http://localhost:3000/
4)Check if the component looks just like the mockup in different screen sizes
